corporate/views/billing_page.py

```diff
@@ -26,6 +26,16 @@
 
 billing_logger = logging.getLogger("corporate.stripe")
 
+ALLOWED_PLANS_API_STATUS_VALUES = [
```

Comment:
I was not able to figure out how to represent this as a `LITERAL`, so couldn't convert endpoints here into `typed_endpoint`.